enigma-principal/app/src/esgx/keys_keeper_u.rs

```diff
@@ -33,11 +33,10 @@ extern "C" {
 /// let response = get_enc_state_keys(enclave.geteid(), request, nonce, None).unwrap();
 /// ```
 #[logfn(DEBUG)]
-pub fn get_enc_state_keys(eid: sgx_enclave_id_t, request: StateKeyRequest, epoch_nonce: U256, sc_addrs: Option<&[ContractAddress]>) -> Result<StateKeyResponse, Error> {
+pub fn get_enc_state_keys(eid: sgx_enclave_id_t, request: StateKeyRequest, epoch_nonce: U256, sc_addrs: &[ContractAddress]) -> Result<StateKeyResponse, Error> {
```

Comment:
why did you remove the option from the addresses? from my understanding, there is still the case in which no addresses are sent to the km (at the beginning of an epoch), isn't it?

Reply:
It is not. This function always gets addresses: or addresses from input, or all epoch addresses.

Reply:
I will fix the test that gets empty vector of addresses. Thank you @AvishaiW, it is a very good catch!